test/systemtests/aci_util.go

```diff
@@ -14,7 +14,7 @@ import (
 // EPSpec for aci-gw
 type EPSpec struct {
 	Tenant string `json:"tenant,omitempty"`
-	App    string `json:"app,omitempty"`
+	App    string `json:"app-prof,omitempty"`
```

Comment:
I needed this small change to work with aci-gw container. I could have opened up another PR. But I asked Sanjeev to incorporate in his PR at that time. This wont cause any problem here.